.circleci/envoy.yaml

```diff
@@ -18,13 +18,14 @@ static_resources:
                         - match: { prefix: "/" }
                           route: { cluster: scalar-ledger }
                       cors:
-                        allow_origin:
-                          - "*"
+                        allow_origin_string_match:
```

Comment:
Can you explain what this change is for ?

Reply:
We use `allow_origin_string_match` to replace `allow_origin` because `allow_origin` became a deprecation in Envoyproxy 1.12.0.
https://www.envoyproxy.io/docs/envoy/v1.12.1/intro/deprecated#version-1-12-0-october-31-2019

The effect is the same.
It allows all potential requests from the browser.